cypress/component/ErrorAlert.cy.tsx

```diff
@@ -0,0 +1,70 @@
+import ErrorAlert from '../../src/components/ErrorAlert';
+import { failedQueryResponse } from '../fixtures/mocked-responses';
+
+const errorTitle = 'errorTitle';
+const explanation = 'something went wrong.';
+const errorMessage = JSON.stringify(failedQueryResponse, null, 2);
```

Comment:
To match the existing tests:

```suggestion
const props = {
    errorTitle : 'errorTitle',
    explanation : 'something went wrong.',
    errorMessage : JSON.stringify(failedQueryResponse, null, 2)
}
```